unlock-app/src/hooks/useKey.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@tanstack/react-query'
+import { useWeb3Service } from '~/utils/withWeb3Service'
+
+interface ValidKeyProps {
+  lockAddress: string
+  network: number
+  account?: string
+}
+
+export const useValidKey = ({
```

Comment:
Please check if we already hae that somewhere?

Reply:
we dont have something similar for this

Reply:
ok